rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+
+Authors must address all comments written by the deadline. This doesn't mean every comment and suggestion must be accepted and incorporated, but they must be carefully read and responded to. Comments written after the deadline may be addressed by the author, but they should be considered as a lower priority.
+
+Every RFC has a lifecycle. The life cycle has the following phases:
+* **Active**: The deadline for comments on this RFC has passed and the authors have decided to go ahead with it. When making the PR, the doc should start out in this state, since it should be active once the PR is accepted.
+* **Dropped**: The authors have decided not to move forward with the changes proposed in this RFC. This might even happen after the proposal was approved.
+* **Superseded**: The changes proposed on this RFC aren't in effect anymore, the document is kept for historical purposes and there is a new RFC that’s more current.
+
+### Approval
+The proposal should be posted with a date by which the author would like to see the approval decision to be made. How much time is given to comment depends on the size and complexity of the proposed changes. Anything between 1 week and 3 weeks seems like a reasonable timespan. Driving the actual decisions should follow the [lazy majority](https://cwiki.apache.org/confluence/display/KAFKA/Bylaws#Bylaws-Approvals) approach. If a discussion is still going strong once it’s approaching the initially set date, the author(s) might consider extending the time for discussion. This should happen explicitly and be announced on the RFC’s PR by updating the PR description accordingly.
+
+### When to write an RFC?
+Writing an RFC should be entirely voluntary. There is always the option of going straight to a pull request. However, for larger changes, it might be wise to de-risk the risk of rejection of the pull request by first gathering input from the community. Therefore it’s up to every member of our community to decide themselves when they want to reach for this tool.
+
+It’s encouraged to write an RFC for any major change. A major change might be:
+Addition of any major new feature or subsystem
+Changes that impact existing, public APIs. This includes Java APIs, but also things like GFSH commands and externally exposed protocols.
+
+### How to write an RFC?
+1. Copy the RFC template (in same folder as this document) and write your proposal! It's up to the author's discretion to decide which section in the template make sense for their proposal. It's recommended to cover the problem the proposal is solving, who it affects, how you’re proposing to solve it, and answers to frequently asked questions. Explicitly listing the goals will also make it easier to retrospect later on the proposal.
+2. Add your RFC to the geode/rfcs source directory, update to the next unique number. 
+3. Post a PR for your RFC putting [RFC] in the title. Make sure to state the deadline clearly in your PR. 
```

Comment:
Can we change this to:
```
3. Post a PR for your RFC prefixing the title with `RFC-X: `, where X is the proposed RFC number. Make sure to state the deadline clearly in your PR. 
```

Reply:
I think we should use the wiki for proposals with a [DISCUSS] thread.  I don't see that a PR is helpful.

Reply:
See https://cwiki.apache.org/confluence/display/GEODE/Project+Proposals+and+Specifications.

Reply:
Can you please qualify how using the wiki is more helpful than the PR process?

The PR process:
* has bidirectional email integration
* has single place to have discussion
* supports inline commenting
* keeps RFC and errata changes documented with code changes in time
* keeps RFC with sources it effects
* Isn't another tool for contributors to use

The Wiki process:
* doesn't have group notification on changes, only followers are notified.
* discussions can happen in email or on the wiki and aren't integrated, so conversation can fragment on two systems, email and wiki.
* can't comment inline
* changes to wiki are tracked independent of code it may effect
* RFCs are not near the code, hiding away in wiki
* contributors have to get separate access and permissions edit wiki, is this possible for non-committers?

Reply:
A few corrections to the above - 

* The wiki can comment inline
* it has threaded discussions as well
* Discussions can be resolved.
* Contributors have access to the wiki
* Wiki changes and comments do send emails

I'll post a separate comment about why I think the wiki is a better place.

Reply:
Thanks for showing me the discussion thing. I didn't get any emails from your comments on the wiki though. Where is that email sent?

Reply:
I think you have to be watching the wiki page to get emails. You can see where your email is sent if you go to your profile in confluence.

Reply:
So here is my problem. If everyone on dev@geode doesn't get an email from the wiki inviting them to review and edit then you must send one manually. If you send one manually people will comment both via the email and via the wiki. Since that email won't feed back into the wiki, like a PR email does, we will have to either look both places for comments or remind the commenter to post those same comments on the wiki. 

When someone comments will it automatically make them a watcher of that document like PR does?

I worry a lot about this because I have seen how commenting about online documents have gone and they have always ended up split across the document and email and keeping all the threads intact is difficult. The priority of the initial RFC process is the C, comments. I believe we need to prioritize that aspect of it. After the commenting phase I really don't care where it ends up.

Reply:
I think the manual email to the dev list is a requirement for this RFC process. PRs don't email the dev list either, so few people are going to see the RFC unless they are watching all PRs and happen to catch that one of them is an RFC.

Reply:
@pivotal-jbarrett I made the change that you requested that started this inline comment thread. Can you confirm that your original concern has been addressed?